dali/operators/audio/nonsilence_op.h

```diff
@@ -228,8 +228,8 @@ class NonsilenceOperatorCpu : public NonsilenceOperator<CPUBackend> {
                   args.reset_interval = reset_interval_;
 
                   auto res = DetectNonsilenceRegion(intermediate_buffers_[thread_id], args);
-                  auto beg_ptr = output_begin[sample_id].mutable_data<int>();
-                  auto len_ptr = output_length[sample_id].mutable_data<int>();
+                  auto beg_ptr = output_begin.mutable_tensor<int>(sample_id);
+                  auto len_ptr = output_length.mutable_tensor<int>(sample_id);
```

Comment:
Nitpick:
```suggestion
                  int *beg_ptr = output_begin.mutable_tensor<int>(sample_id);
                  int *len_ptr = output_length.mutable_tensor<int>(sample_id);
```
It's not like we have a long or complicated type here.